web/template/layout/mlpa-case-summary.gohtml

```diff
@@ -13,13 +13,35 @@
 
         <div class="govuk-grid-row" data-id="case-summary" aria-expanded="true">
             <div class="govuk-grid-column-one-third">
-                <svg class="app-svg-icon--inline" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 -2 19 19" overflow="visible">
-                    <circle cx="9.5" cy="9.5" r="9.5" fill="#1D70B8"/>
-                    <circle cx="9.5" fill="#fff" cy="13.8" r="1.3"/>
-                    <path fill="#fff" d="M8.4 4.5h2.2l-.2 7H8.6l-.2-7Z"/>
-                </svg>
-                <h3 class="govuk-heading-s govuk-!-display-inline">Warnings</h3>
-                <p class="govuk-body">There are none in place.</p>
+                <div class="govuk-!-margin-bottom-4" data-role="warnings-list">
```

Comment:
Nit, but I don't think this attribute is doing anything:
```suggestion
                <div class="govuk-!-margin-bottom-4">
```

Reply:
I use it in the cypress tests. I can remove it an make the cypress tests more gnarly instead.

Reply:
actually, I duplicated that anyway, I'll just remove it

Reply:
Ah sorry, totally missed that 🤦🏽